pkgs/development/python-modules/napalm/default.nix

```diff
@@ -17,42 +46,31 @@ buildPythonPackage rec {
     hash = "sha256-JqjuYMJcP58UMn1pPYg7x8KpqCKQUs19Ng9HbI2iX38=";
   };
 
-  patches = [
-    # netmiko 4.0.0 support
-    (fetchpatch{
-      url = "https://github.com/napalm-automation/napalm/commit/4b8cc85db3236099a04f742cf71773e74d9dd70e.patch";
-      excludes = [ "requirements.txt" ];
-      hash = "sha256-DBKp+wiKd+/j2xAqaQL3UCcGQd6wnWcNTsNXBBt9c98=";
-    })
-    (fetchpatch{
-      url = "https://github.com/napalm-automation/napalm/commit/4a8b5b1823335dd79aa5269c038a1f08ecd35cdd.patch";
-      hash = "sha256-uiou/rzmnFf4wAvXwmUsGJx99GeHWKJB2JrMM1kLakM=";
-    })
+  nativeBuildInputs = [
+    setuptools
   ];
 
-  postPatch = ''
-    substituteInPlace requirements.txt \
-      --replace "netmiko>=3.3.0,<4.0.0" "netmiko"
-  '';
-
   propagatedBuildInputs = [
     cffi
-    paramiko
-    requests
     future
-    textfsm
-    invoke
     jinja2
+    junos-eznc
+    lxml
+    ncclient
     netaddr
-    pyyaml
-    pyeapi
     netmiko
-    junos-eznc
-    ciscoconfparse
+    # breaks infinite recursion
+    (netutils.override { napalm = null; })
```

Comment:
This sparked my eye because normally we would do that in python-packages.nix but since we got the hook that catches conflicts that might be okay.